services/blockchain-indexer/package.json

```diff
@@ -37,7 +37,7 @@
     "bull": "^4.8.1",
     "camelcase": "^6.3.0",
     "ioredis": "^4.28.5",
-    "lisk-service-framework": "https://github.com/LiskHQ/lisk-service/raw/b0b3afb0b93644276cfa98f1a49747dc1ad97fbe/framework/dist/lisk-service-framework-1.4.15.tgz",
+    "lisk-service-framework": "https://github.com/LiskHQ/lisk-service/raw/1188-improve-MySQL-implementation/framework/dist/lisk-service-framework-1.4.16.tgz",
```

Comment:
Let's please stick to using the commit hashes. I'm unsure, if this would still work after we delete these branches.